docs/readthedocs/source/doc/Nano/Overview/nano.md

```diff
@@ -0,0 +1,102 @@
+# Nano User Guide
+
+## **1. Overview**
+
+BigDL Nano is a python package to transparently accelerate PyTorch and TensorFlow applications on Intel hardware. It provides a unified and easy-to-use API for several optimization techniques and tools, so that users can only apply a few lines of code changes to make their PyTorch or TensorFlow code run faster.
+
+---
+## **2. Get Started**
+
+### **2.1 PyTorch**
+
+#### **2.1.1 Install**
+
+BigDL-Nano can be installed using pip and we recommend installing BigDL-Nano in a conda environment.
+
+```bash
+pip install bigdl-nano[pytorch]
+```
+
+After installing bigdl-nano, you can run the following command to setup a few environment variables. 
+
+```bash
+source bigdl-nano-init
+```
+
+The `bigdl-nano-init` scripts will export a few environment variable according to your hardware to maximize performance. 
+
+In a conda environment, this will also add this script to `$CONDA_PREFIX/etc/conda/activate.d/`, which will automaticly run when you activate your current environment.
+
+In a pure pip environment, you need to run `source bigdl-nano-init` every time you open a new shell to get optimal performance.
```

Comment:
Should we teach our users how to unset those variables (`source bigdl-nano-unset-env`)?

Reply:
good catch!